src/bootstrap/defaults/bootstrap.ferrocene-dist.toml

```diff
@@ -5,6 +5,8 @@
 # recommended to run this configuration during local development, except if your development
 # configuration cannot reproduce a CI failure.
 
+change-id = "ignore"
```

Comment:
I don't think we should ignore the changelog, see https://ferrous.zulipchat.com/#narrow/channel/385800-Ferrocene/topic/bootstrap.20changelog/with/513801251

Reply:
@tshepang can we skip this change?